docs/guide/etl.rst

```diff
@@ -564,9 +564,10 @@ Extension
 ---------
```

Comment:
I think its the JSON code block above is not closed property, with missing '}' ?